### PR TITLE
Q215 Add KthLargestElement.py

### PR DESCRIPTION
**Pull request by peinanteng**

### CODE REVIEW

File: KthLargestElement.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+                nums[left], nums[i] = nums[i], nums[left]
+                i += 1
+            elif nums[i] < pivot:
+                right -= 1
+                nums[right], nums[i] = nums[i], nums[right]
+            else:
+                i += 1
+        if left >= k - 1:
+            return self.quickSelect(nums[:left + 1], k)
+        elif right <=  k - 1:
```
Comment (nkukarl) [inline KthLargestElement.py]: You don't need `elif` here

File: KthLargestElement.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            return self.quickSelect(nums[right:], k - right)
+        else:
```
Comment (nkukarl) [inline KthLargestElement.py]: You don't need `else` here

File: KthLargestElement.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+class Solution(object):
+    def findKthLargest(self, nums, k):
+        """
+        :type nums: List[int]
+        :type k: int
+        :rtype: int
+        """
+        return self.quickSelect(nums, k)
```
Comment (nkukarl) [inline KthLargestElement.py]: Do you really need to define `quickSelect`?
Guess you can just recursively call `findKthLargest` to do the same thing?

File: KthLargestElement.py (hunk 1 above)
Comment (peinanteng) [reply KthLargestElement.py]: I guess I need elif here

File: KthLargestElement.py (hunk 3 above)
Comment (peinanteng) [reply KthLargestElement.py]: Comments has been addressed.

File: KthLargestElement.py (hunk 2 above)
Comment (peinanteng) [reply KthLargestElement.py]: Sure, I agree

**Pull request merged**